src/main/java/org/logstash/dissect/fields/NormalField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class NormalField extends AbstractField {
+
+    private static final Field missing = new NormalField("missing field", 100000);
+    public static Field getMissing() {
+        return missing;
+    }
+
+    public static Field create(String s) {
+        return new NormalField(s);
+    }
+
+    private NormalField(String s) {
+        super(s, 1);
+    }
+
+    private NormalField(String s, int ord) {
+        super(s, ord);
+    }
+
+    @Override
+    public boolean saveable() {
+        return true;
+    }
+
+    @Override
+    public void append(Map<String, Object> keyValueMap, ValueResolver values) {
+        keyValueMap.put(this.name, values.get(this));
+    }
+
+    @Override
+    public void append(Event event, ValueResolver values) {
+      event.setField(this.name, values.get(this));
```

Comment:
FYI - this plugin, like all the others, is independently published to Rubygems (like Maven Central). It is common practice in LS to release a MVP at 1.0.0 and quickly iterate to a more complete solution. A new version of LS is not needed for each plugin release.